src/components/data-management/ProjectDetails.jsx

```diff
@@ -1,4 +1,5 @@
 import React, { useState, useEffect, useRef } from 'react';
+import _ from 'lodash';
```

Comment:
I don't see where `_` is used. I think you can remove this.

Reply:
When clicking `add metadata` a new column is added to the table. But when uploading a file the table is updated all the time and you can see that the old changes were setting the table columns to the initial ones every time which removes the metadata popover.